style.css

```diff
@@ -741,10 +765,79 @@ textarea::placeholder {
   bottom: -1.63%;
 }
 
-@media screen and (max-width: 768px) {
-  .mobile-hide {
-    display: none;
-  }
+/* MOBILE MENU */
+
+.close-fa-icon {
+  display: none;
+}
+
+.mobile-menu-display-none {
+  display: none;
+}
+
+.mobile-menu-display-content {
+  display: contents;
+}
+
+.list-menu-position {
+  z-index: 30;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  top: 106px;
+  left: 26px;
+  list-style: none;
+}
+
+.list-menu-items-font {
+  position: relative;
+  z-index: 30;
+  font-family: 'Poppins', sans-serif;
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 44px;
+  color: #fff;
+  text-decoration: none;
+}
```

Comment:
- Kindly, add some space to the menu lists and move the close button a little bit to the top to stick to the Figma design and have too better UI 👌
![2023-03-15_08-48](https://user-images.githubusercontent.com/52010789/225221481-d789e72f-2582-4ac5-9d71-deead5ec4912.png)